plenum/server/node.py

```diff
@@ -247,15 +241,20 @@ def __init__(self,
         self.monitor_init(pluginPaths)
         self.replicas = self.create_replicas()
 
+        self.requiredNumberOfInstances = 0
         # Any messages that are intended for protocol instances not created.
         # Helps in cases where a new protocol instance have been added by a
         # majority of nodes due to joining of a new node, but some slow nodes
         # are not aware of it. Key is instance id and value is a deque
         # TODO is it possible for messages with current view number?
         self.msgsForFutureReplicas = {}
 
+        # 3PC state consistency watchdog based on network events
+        self.network_i3pc_watcher = NetworkInconsistencyWatcher(self.on_inconsistent_3pc_state_from_network)
```

Comment:
Do we still need this?

Reply:
Good question, but I think out-of-scope of this PR